File: v2/java/src/main/java/vestorly/client/model/Group.java
package vestorly.client.model;


import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Group  {
  
  private String id = null;
  private String name = null;
  private Boolean isDefault = null;
  private Boolean isHidden = null;
  private String newWeeklyMailerContent = null;
  private String newsletterSubject = null;
  private Boolean autopublish = null;
  private Integer numberArticlesPerGroup = null;
  private Integer numberArticlesPerNewsletter = null;

  
  /**
   **/
  @ApiModelProperty(required = true, value = "")
  @JsonProperty("_id")
  public String getId() {
    return id;
  }
  public void setId(String id) {
    this.id = id;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("name")
  public String getName() {
    return name;
  }
  public void setName(String name) {
    this.name = name;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("is_default")
  public Boolean getIsDefault() {
    return isDefault;
  }
  public void setIsDefault(Boolean isDefault) {
    this.isDefault = isDefault;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("is_hidden")
  public Boolean getIsHidden() {
    return isHidden;
  }
  public void setIsHidden(Boolean isHidden) {
    this.isHidden = isHidden;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("new_weekly_mailer_content")
  public String getNewWeeklyMailerContent() {
    return newWeeklyMailerContent;
  }
  public void setNewWeeklyMailerContent(String newWeeklyMailerContent) {
    this.newWeeklyMailerContent = newWeeklyMailerContent;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("newsletter_subject")
  public String getNewsletterSubject() {
    return newsletterSubject;
  }
  public void setNewsletterSubject(String newsletterSubject) {
    this.newsletterSubject = newsletterSubject;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("autopublish")
  public Boolean getAutopublish() {
    return autopublish;
  }
  public void setAutopublish(Boolean autopublish) {
    this.autopublish = autopublish;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("number_articles_per_group")
  public Integer getNumberArticlesPerGroup() {
    return numberArticlesPerGroup;
  }
  public void setNumberArticlesPerGroup(Integer numberArticlesPerGroup) {
    this.numberArticlesPerGroup = numberArticlesPerGroup;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("number_articles_per_newsletter")
  public Integer getNumberArticlesPerNewsletter() {
    return numberArticlesPerNewsletter;
  }
  public void setNumberArticlesPerNewsletter(Integer numberArticlesPerNewsletter) {
    this.numberArticlesPerNewsletter = numberArticlesPerNewsletter;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Group {\n");
    
    sb.append("  id: ").append(id).append("\n");
    sb.append("  name: ").append(name).append("\n");
    sb.append("  isDefault: ").append(isDefault).append("\n");
    sb.append("  isHidden: ").append(isHidden).append("\n");
    sb.append("  newWeeklyMailerContent: ").append(newWeeklyMailerContent).append("\n");
    sb.append("  newsletterSubject: ").append(newsletterSubject).append("\n");
    sb.append("  autopublish: ").append(autopublish).append("\n");
    sb.append("  numberArticlesPerGroup: ").append(numberArticlesPerGroup).append("\n");
    sb.append("  numberArticlesPerNewsletter: ").append(numberArticlesPerNewsletter).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/vestorly/Model/Group.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace vestorly.model {
  [DataContract]
  public class Group {
    
    
    [DataMember(Name="_id", EmitDefaultValue=false)]
    public string id { get; set; }

    
    
    [DataMember(Name="name", EmitDefaultValue=false)]
    public string name { get; set; }

    
    
    [DataMember(Name="is_default", EmitDefaultValue=false)]
    public bool? isDefault { get; set; }

    
    
    [DataMember(Name="is_hidden", EmitDefaultValue=false)]
    public bool? isHidden { get; set; }

    
    
    [DataMember(Name="new_weekly_mailer_content", EmitDefaultValue=false)]
    public string newWeeklyMailerContent { get; set; }

    
    
    [DataMember(Name="newsletter_subject", EmitDefaultValue=false)]
    public string newsletterSubject { get; set; }

    
    
    [DataMember(Name="autopublish", EmitDefaultValue=false)]
    public bool? autopublish { get; set; }

    
    
    [DataMember(Name="number_articles_per_group", EmitDefaultValue=false)]
    public int? numberArticlesPerGroup { get; set; }

    
    
    [DataMember(Name="number_articles_per_newsletter", EmitDefaultValue=false)]
    public int? numberArticlesPerNewsletter { get; set; }

    
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Group {\n");
      
      sb.Append("  id: ").Append(id).Append("\n");
      
      sb.Append("  name: ").Append(name).Append("\n");
      
      sb.Append("  isDefault: ").Append(isDefault).Append("\n");
      
      sb.Append("  isHidden: ").Append(isHidden).Append("\n");
      
      sb.Append("  newWeeklyMailerContent: ").Append(newWeeklyMailerContent).Append("\n");
      
      sb.Append("  newsletterSubject: ").Append(newsletterSubject).Append("\n");
      
      sb.Append("  autopublish: ").Append(autopublish).Append("\n");
      
      sb.Append("  numberArticlesPerGroup: ").Append(numberArticlesPerGroup).Append("\n");
      
      sb.Append("  numberArticlesPerNewsletter: ").Append(numberArticlesPerNewsletter).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}
